config/oidc-auth.php

```diff
@@ -57,13 +57,18 @@
     | JWT claims in id_token required to authenticate. Arrays set required
     | elements in an array. Other values are matched exactly.
     |
-    | This can also be a Closure to check id_token. id_token will be passed as
-    | the first parameter. Return true to indicate a pass or false for a fail.
+    | You can provide a \Closure or
+    | \Laravel\SerializableClosure\SerializableClosure that accepts id_token as
+    | its argument to perform customized validation.
+    | Return true to indicate a pass or false for a fail.
+    | When passing SerializableClosure, use `APP_KEY` environment variable to sign
```

Comment:
i don't think this needs to be specifically noted, since a laravel application without `APP_KEY` (or actually, `app.key` config) won't work properly anyway

instead, i think the fact that the serialized SerializableClosure approach is to make the config compatible with config cache while still provides the flexibility of a closure, should be noted

Reply:
It is explicitly noted because users should manually set the secret, or it would cause error when CallbackController unserializes data.